lib/helpers/rule-helper.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Base for all rules
+ */
+'use strict';
+
+function RuleHelper(context) {
```

Comment:
Sorry that comment was left over form when I first made the file. Originally I was going to do an inheritance model but I thought a generic helper would make more sense in this case. If you want I can re-factor and make all the rules inherit from a BaseRule.